src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenNullCoalescingAssignmentTests.cs

```diff
@@ -1418,33 +1418,26 @@ public static void Main()
         [Fact]
         public void TypeParameterLHS()
         {
-            CreateCompilation(@"
-class C
-{
-    void M<T>(T t)
-    {
-        t ??= default;
```

Comment:
>t ??= default [](start = 8, length = 13)

Is this a success case now? Do we have a test that verifies that the behavior (and perhaps IL) is what we expect?